packages/market-service/src/utils/rateLimiters.ts

```diff
@@ -1,23 +1,7 @@
-import type { AxiosInstance } from 'axios'
-import axios from 'axios'
-import rateLimit from 'axios-rate-limit'
 import { pRateLimit } from 'p-ratelimit'
```

Comment:
yearn-vaults is the only thing using this at this point. was that intentional? or remove from yearn-vault and get rid of this util and associated tests